server/services/overviewPageService.ts

```diff
@@ -32,6 +34,75 @@ export default class OverviewPageService {
     }
   }
 
+  private getPersonalDetails(prisonerData: Prisoner): PersonalDetails {
+    const personalDetailsMain = [
```

Comment:
Could extract this into a mock object so it can also be used for unit and e2e testing.